server/index.js

```diff
@@ -1,13 +1,23 @@
 var express = require('express');
 var parser = require('body-parser');
 var db = require('../db/index.js');
+var path = require('path');
 
 let app = express();
 app.use(parser.json());
 app.use(express.static(__dirname + '/../react-client/dist'));
 
+app.get('/:id', (req, res) => {
+  res.sendFile(path.resolve('react-client/dist/index.html'));

```

Comment:
As we discussed, this will eventually be productName instead of id, but this is a great piece of functionality to iron out before making that switch.  nice work